universal-application-tool-0.0.1/app/views/BaseHtmlLayout.java

```diff
@@ -20,18 +20,21 @@
  * #render(HtmlBundle)} method.
  */
 public class BaseHtmlLayout {
+  private static final String STAGING_HOST_NAME = "staging.seattle.civiform.com";
   private static final String TAILWIND_COMPILED_FILENAME = "tailwind";
   private static final String[] FOOTER_SCRIPTS = {"main", "radio", "toast"};
   private static final String BANNER_TEXT =
       "Do not enter actual or personal data in this demo site";
 
   public final ViewUtils viewUtils;
   private final String measurementId;
+  private final String hostName;
 
   @Inject
   public BaseHtmlLayout(ViewUtils viewUtils, Config configuration) {
     this.viewUtils = checkNotNull(viewUtils);
     this.measurementId = checkNotNull(configuration).getString("measurement_id");
+    this.hostName = checkNotNull(configuration).getString("base_url");
```

Comment:
Suggestion (blocking): this would include the schema (i.e. https), please use `URI.create(baseUrl).getHost();` to get the hostname from the base url.

Reply:
fixed